### ISSUE TITLE
Can't boot DS3622+ from new kernel

### ISSUE THREAD

**Issue by SadykovIV** [pocopico/tinycore-redpill#79]
[boot-log.txt](https://github.com/pocopico/tinycore-redpill/files/8751675/boot-log.txt)
![2022-05-23_08-23-13](https://user-images.githubusercontent.com/86406357/169749527-db4e0a95-ad13-43e1-b378-ce9e5f63b2c1.png)

**Comment by SadykovIV** [comment]
It used to work. But after the last script update, I can't start the VM
The old bootloader allows to start the system, does not let me install it

**Comment by pocopico** [comment]
Yes, i should update some loader files for jumkey last changes.

**Comment by donbaba1** [comment]
Exactly, the same issue I am having, the system is going into a recovery mode loop.

**Comment by pocopico** [comment]
After install, The system will fetch latest small update from syno and apply it. If you encounter the recovery loop then you need to boot into TCRP and run 

./rploader.sh update 
./rploader.sh postupdate <your platform is>

**Comment by donbaba1** [comment]
[installlogs.txt](https://github.com/pocopico/tinycore-redpill/files/8837831/installlogs.txt)

brand new install, after updating with the .pat file from syno, i rebooted in the TCRP, run the commands as you suggested and rebooted again.. it does not shows up in the network after that. 

note : doing this on ESXI - VMware machine.

**Comment by pocopico** [comment]
Can you please check the serial console and maybe post the messages ?

**Comment by donbaba1** [comment]
I don’t have physical machine. It’s a virtual machine. I was checking the console screen and it’s sitting at booting kernel n that’s all

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by donbaba1** [comment]
![image](https://user-images.githubusercontent.com/45697108/172015922-83266052-1c55-41fb-b412-c0e905b0d5c1.png)

**Comment by pocopico** [comment]
> Can you please check the serial console and maybe post the messages ?

And you remember that in a different state ?

**Comment by donbaba1** [comment]
attached is the serial output. captured for 5 mins after the boot.
[serial.txt](https://github.com/pocopico/tinycore-redpill/files/8838312/serial.txt)

**Comment by pocopico** [comment]
So it looks that it’s booting ok. If you login with root and no pass do you get a prompt? Can you cat /var/log/junio_reason*

**Comment by donbaba1** [comment]
Not sure its right to ask in this forum, would you like to connect to the machine remotely or something.. to speed up ?

**Comment by donbaba1** [comment]
I am sorry, as i dont know how to connect to serial port over network in esxi

**Comment by donbaba1** [comment]
SSH / Telnet over "Network serial" not working,..

Can i login from the vmware console ? if yes.. how ?

**Comment by pocopico** [comment]
https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-C6FBCF66-5796-4EE6-BF47-4DCAA9DCD1E3.html

**Comment by donbaba1** [comment]
![image](https://user-images.githubusercontent.com/45697108/172019416-579d883f-3464-4b8e-9571-407a5f868caf.png)

**Comment by donbaba1** [comment]
I know this is not helping you, not sure what else i can help you. i want to help you to get this solved.

**Comment by pocopico** [comment]
You need to enable the port on the firewall of your esxi host

**Comment by donbaba1** [comment]
Perfect, that did the trick :). learned something new on the esxi today :) :) . wonderfull. 

Attaching the file for your review. 

But i dont see  anything in the junio_reason* in the folder /var/log/ ...

[putty.log](https://github.com/pocopico/tinycore-redpill/files/8838573/putty.log)

**Comment by donbaba1** [comment]
![image](https://user-images.githubusercontent.com/45697108/172023663-589b417c-d58a-42f0-aaef-e1585aef550e.png)

**Comment by donbaba1** [comment]
![image](https://user-images.githubusercontent.com/45697108/172023728-42625770-8148-47a2-a31a-7e96aed7ced2.png)



Not sure about these errors. just reading through them and pasting :)

**Comment by pocopico** [comment]
So your system looks that is booting fine. Don’t worry about these messages, they are expected. 

Can you check with ifconfig to see if any interfaces are listed or not ?

**Comment by donbaba1** [comment]
![image](https://user-images.githubusercontent.com/45697108/172031349-5b7c6430-a398-44c1-8a7e-8bcbeba5b0c4.png)

**Comment by donbaba1** [comment]
Some progress here... 

I done ahead and disconnected the VNIC on the VM and connected back. it picked up the DHCP IP, but in less than 30 Seconds it powered down the VM,... 
[serial (2).txt](https://github.com/pocopico/tinycore-redpill/files/8839277/serial.2.txt)

**Comment by donbaba1** [comment]
OKie, i redid everything from scrach and its working like charm. 

But one issue  i  am having is the Serial number, ... not able to register it on the webiste.... have generated multiple ones.. still the same issue

**Comment by pocopico** [comment]
You cannot register a non genuine box to syno

**Comment by donbaba1** [comment]
Thank you for all your amazing work. Truely Apprecaite.






Thanks & Regards
Ashish Agarwal

[http://www.linkedin.com/img/webpromo/btn_viewmy_160x33.png]
<http://ca.linkedin.com/in/ashishagarwal44943>
________________________________
